yolact.py

```diff
@@ -1,380 +1,28 @@
 import torch, torchvision
 import torch.nn as nn
 import torch.nn.functional as F
+import torch.backends.cudnn as cudnn
 from torchvision.models.resnet import Bottleneck
 import numpy as np
-from itertools import product
-from math import sqrt
-from typing import List
-from collections import defaultdict
 
-from data.config import cfg, mask_type
+from data.config import mask_type
 from layers import Detect
 from layers.interpolate import InterpolateModule
 from backbone import construct_backbone
 
-import torch.backends.cudnn as cudnn
 from utils import timer
 from utils.functions import MovingAverage, make_net
+from data.config import Config, mask_type
+
+#locally defined modules
+from layers.modules.prediction import PredictionModule
```

Comment:
I've refactored the `yolact.py` file and separated the `nn.Module` classes into standalone `layers/modules/`: FPN, Concat, FastMaskIoU, etc

Reply:
I like this change, been meaning to do it for a while.